tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/otextension/BristolOtShared.java

```diff
@@ -0,0 +1,44 @@
+package dk.alexandra.fresco.tools.ot.otextension;
+
+import dk.alexandra.fresco.framework.network.Network;
+import dk.alexandra.fresco.framework.util.Drbg;
+
+/**
+ * Superclass containing the common variables and methods for the sender and
+ * receiver parties of a Bristol OT extension.
+ */
+public class BristolOtShared {
```

Comment:
These superclasses are quite weird. They serve to shorthand resource access. In this case, there is an implicit resourcepool within the Rot.
I propose to remove all Shared interfaces and replace them with local copies of the resourcepool and network.
This can be further discussed and developed, so I will pause my review here.

Reply:
I have removed the variable access part of the shared classes and in effect deleted a few shared classes. However, where there was shared methods between the receiver and sender I have kept the shared classes in order to avoid code-copy.